Documentation/Installation/ReleaseIntegrity.rst

```diff
@@ -87,9 +87,9 @@ These hashes must match the hashes published on get.typo3.org to ensure package
 Checking file signatures
 ------------------------
 
-TYPO3 uses `Pretty Good Privacy`_ to sign release packages and Git release tags.
+TYPO3 uses ``Pretty Good Privacy``_ to sign release packages and Git release tags.
```

Comment:
```suggestion
TYPO3 uses `Pretty Good Privacy`_ to sign release packages and Git release tags.
```

These are actually properly formatted external links, see the URLs at the bottom.